frontendActions/src/main/resources/GitMacheteBundle.properties

```diff
@@ -6,6 +6,10 @@ action.GitMachete.description.disabled.undefined.machete-branch={0} disabled due
 GitMachete.FetchBackgroundable.notification.fail=Fetch of refspec <b>{0}</b> failed
 GitMachete.FetchBackgroundable.notification.success=Fetch of refspec <b>{0}</b> succeeded
 
+GitMachete.PullBackgroundable.notification.message.view-commits=View Commits
```

Comment:
`.message` part is likely redundant

Reply:
I think it can be useful in some cases to distinguish between `notification.message` and `notification.title`, so maybe this is the reason.

Reply:
But ok, for now we can omit this `message`

Reply:
> I think it can be useful in some cases to distinguish between `notification.message` and `notification.title`, so maybe this is the reason.

this ^